xdr/Stellar-transaction.x

```diff
@@ -335,7 +335,7 @@ union Memo switch (MemoType type)
 case MEMO_NONE:
     void;
 case MEMO_TEXT:
-    string text<28>;
+    opaque text<28>;
```

Comment:
Where do we keep the source of truth for our .x files? Does this get updated there also so that all other SDKs also get updated?

Reply:
I think they are kept here https://github.com/stellar/stellar-core/tree/master/src/xdr

I'm waiting for confirmation in https://github.com/stellar/go/issues/2022 that we're going to change the type of the memo text field. Once we have confirmation we'll need to update the .x files in stellar core and in the go monorepo

Reply:
Files in `xdr` should be exactly the same as in stellar-core repo, we shouldn't change them here nor autogenerated files in: `src/test/java/org/stellar/sdk/xdr`.

Reply:
@bartekn 

> I'm waiting for confirmation in stellar/go#2022 that we're going to change the type of the memo text field. Once we have confirmation we'll need to update the .x files in stellar core and in the go monorepo

Reply:
If this is merged after the changes are in stellar-core master then 👍.